source/_integrations/slide.markdown

```diff
@@ -45,3 +47,5 @@ invert_position:
   default: false
   type: boolean
 {% endconfiguration %}
+
+NOTE: Starting September 2023 an API limit is in place. A scan_interval of at least 300 (600 if you have multiple slides) is needed.
```

Comment:
This is not a thing we should fix or work around in documentation. Home Assistant should have correct and sane default out of the box.

Either open a PR to adjust the default or raise an issue in our Core issue tracker.

Thanks 👍 

../Frenck